### ISSUE TITLE
Is there a way to clear custom additions to the "Fast Filtering" "Subfolder" and "Mask" menus?

### ISSUE THREAD

**Issue by MickeyMcNickname** [downthemall/downthemall#432]
Sometimes it gets busy in there with my own custom attempts which may include many similar looking failures. So I would like to be able to delete one or even all of the custom adds. 

Also, and I think it would be useful if there was a way to indicate a boolean NOT for the "Fast Filtering" field,  or it you added a check box for "Disable These" to go with the checkbox for "Disable Others."

I know you have so many requests and bug reports to work, and I'm not an experienced person with Regex and such. So if there are other ways to do these things, please advise when you have the time.

**Comment by nmaier** [comment]
I think the first part is tracked in #278

>Also, and I think it would be useful if there was a way to indicate a boolean NOT for the "Fast Filtering" field, or it you added a check box for "Disable These" to go with the checkbox for "Disable Others."

Just to clarify, you'd want a the fast filtering filter to then unmatch/unselect everything that matches it?

**Comment by MickeyMcNickname** [comment]
Yes, I know it sounds weird. But I would like to disable some matches.

I can give this example: sometimes there are pictures where both the full-size and thumbnails have the same name, but some other thing that separates them. So lets say that's "tn" for the thumbnail directory. I could match on that extra element and then select "disable others" to get them out of the list of jpegs. Of course this may be my issue only, and other users have found an easier way.
